packages/financial-templates-lib/src/price-feed/UniswapPriceFeed.js

```diff
@@ -26,14 +39,26 @@ class UniswapPriceFeed extends PriceFeedInterface {
     this.getTime = getTime;
     this.historicalLookback = historicalLookback;
     this.invertPrice = invertPrice;
+    // The % of the lookback window (historicalLookback + twapLength) that we want to query for Uniswap
+    // Sync events. For example, 1.1 = 110% meaning that we'll look back 110% * (historicalLookback + twapLength)
+    // seconds, in blocks, for Sync events.
+    this.bufferBlockPercent = 1.1;
+
+    // TODO: Should/Can we read in `poolDecimals` from the this.uniswap?
+    this.poolPrecision = poolDecimals;
+    this.decimals = priceFeedDecimals;
 
     // Helper functions from web3.
     this.toBN = this.web3.utils.toBN;
     this.toWei = this.web3.utils.toWei;
+
+    // Convert _bn precision from poolDecimals to desired decimals by scaling up or down based
+    // on the relationship between poolPrecision and the desired decimals.
+    this.convertDecimals = ConvertDecimals(this.poolPrecision, this.decimals, this.web3);
```

Comment:
nit: can you rename this method `convertPoolDecimalsToPriceFeedDecimals`? I know it's a long function name but `convertDecimals` is too generic I think & I want us to be as consistent as possible within the feeds when talking about decimal conversion. For example, the crypto watch refers to `convertPriceFeedDecimals` so this will be consistent with that naming convention.

Reply:
Done and made the same change for `BalancerPriceFeed`